include/mbedtls/platform.h

```diff
@@ -52,34 +52,36 @@ extern "C" {
 #include <time.h>
 #if !defined(MBEDTLS_PLATFORM_STD_SNPRINTF)
 #if defined(_WIN32)
-#define MBEDTLS_PLATFORM_STD_SNPRINTF   mbedtls_platform_win32_snprintf /**< Default snprintf to use  */
+#define MBEDTLS_PLATFORM_STD_SNPRINTF   mbedtls_platform_win32_snprintf 
+/**< The default \c snprintf function to use.  */
```

Comment:
Does Doxygen digest the line break here?

Reply:
Yes - I produced to verify before I did this (in several files)

Reply:
When the comment isn't on the same line, please put it on the line *before*, not *after* (and thus starting with `/**` and not `/**<`). This convention is pretty much universally followed in programming: comments go before or on the same line as what they're commenting, not after.

Reply:
returned to the same line to keep consistent with the rest of your defines. They all exceed 80 chars.